### PR TITLE
BUNDLE-2683: VBE Braintree - Error occurs in Admin Braintree Settlement Report

### PR DESCRIPTION
**Pull request by magterskine**
### Description:
https://jira.corp.magento.com/browse/MCLOUD-6684:
Adds patch to fix issue for Braintree VBE where Admin viewing Braintree Settlement Report causes error screen

### Fixed Issues:
https://jira.corp.magento.com/browse/BUNDLE-2683

### Manual Testing Scenarios
1. Enable Braintree
2. Navigate to: Magento Admin Panel > Reports > Braintree Settlement
3. “Braintree Settlement Report” page should be loaded

### CODE REVIEW

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
The is no proper description for the PR - please add one with correct format and full description
Tests are failing
PR name has type (BUNDLE not BUNLDE)

**Comment by viktym** [comment]
Please, fix patch filename (use __ after ticket number and before the version):
```
ERROR: The command "php ./vendor/bin/ece-patches apply --no-interaction" failed.                                                                                                     
  [Magento\CloudPatches\Patch\Collector\CollectorException]                                         
  The patch filename 'BUNDLE-2683_braintree_settlement_report_fix_4.1.0.patch' has invalid format.  
  Correct format: <TICKET_NUMBER>__<TITLE>__<PACKAGE_VERSION>.patch                                 
  Example: MAGECLOUD-2899__fix_redis_slave_configuration__2.3.0.patch  
```

**Comment by shiftedreality** [comment]
@viktym @magterskine please re-create this PR into https://github.com/magento/quality-patches

File: patches/BUNDLE-2683__braintree_settlement_report_fix__4.0.1.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+diff --git a/vendor/paypal/module-braintree-core/Ui/Component/Report/Listing/Column/PaymentType.php b/vendor/paypal/module-braintree-core/Ui/Component/Report/Listing/Column/PaymentType.php
```
Comment (shiftedreality) [inline patches/BUNDLE-2683__braintree_settlement_report_fix__4.0.1.patch]: Please use `diff -Naur`

**Review by shiftedreality — CHANGES_REQUESTED** [review]

**Review by NadiyaS — DISMISSED** [review]

**Review by shiftedreality — DISMISSED** [review]

**Review by BarnyShergold — DISMISSED** [review]

File: patches.json [hunk 2]
```diff
@@ -335,6 +335,11 @@
       "3.4.1": "BUNDLE-2554__set_payment_info_bug_fix__3.4.1.patch"
     }
   },
+  "paypal/module-braintree-core": {
+    "Fix Braintree Settlement report error": {
+      "4.1.0": "BUNDLE-2683__braintree_settlement_report_fix__4.1.0.patch"
```
Comment (oshmyheliuk) [inline patches.json:340]: The patch file path is different to real file:
`BUNDLE-2683__braintree_settlement_report_fix__4.1.0.patch`
vs
`BUNDLE-2683__braintree_settlement_report_fix__4.0.1.patch`
Please change

**Review by oshmyheliuk — CHANGES_REQUESTED** [review]

File: patches.json (hunk 2 above)
Comment (magterskine) [reply patches.json:340]: Thanks. 4.1.0 is correct.

**Review by NadiyaS — APPROVED** [review]

**Review by BarnyShergold — APPROVED** [review]

**Review by oshmyheliuk — APPROVED** [review]

**Review by shiftedreality — APPROVED** [review]

**Comment by mveeramneni** [comment]
QA Approved

**Pull request merged**